src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx

```diff
@@ -21,11 +21,13 @@ import Spinner from '../styled/spinner';
 // This wraps Control + ControlError message
 // * can NOT be a flex box have because of position: absolute on "control error message"
 // * can NOT have overflow hidden because "control error message" overflows
-const FormFieldControlErrorWrapper = styled(Box)`
+const FormFieldControlErrorWrapper = styled(({inline, ...props}) => <Box {...props} />)`
```

Comment:
This fixes props being passed to native DOM elements

Reply:
👍